community/logging/src/main/java/org/neo4j/logging/FormattedLog.java

```diff
@@ -203,19 +207,30 @@ protected void writeLog( PrintWriter out, String message )
         @Override
         protected void writeLog( PrintWriter out, String message, Throwable throwable )
         {
+            requireNonNull( message, "message" );
+
             lineStart( out );
             out.write( message );
-            out.write( ' ' );
-            out.write( throwable.getMessage() );
-            out.println();
-            throwable.printStackTrace( out );
+            if ( throwable != null )
+            {
+                if ( throwable.getMessage() != null )
```

Comment:
I wonder if this should also be handled in `AbstractPrintWriterLogger`, as the end result is the same as logging only a message - so it could call `#writeLog( PrintWriter out, String message )` instead. However, I was actually thinking it might be better to log at least the class name?